target/linux/ath79/dts/ar7240_buffalo_whr-g301n.dts

```diff
@@ -0,0 +1,188 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar7240.dtsi"
+
+/ {
+	compatible = "buffalo,whr-g301n", "qca,ar7240";
+	model = "Buffalo WHR-G301N";
+
+	aliases {
+		led-status = &router;
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
+
+		router_on {
+			label = "router_on";
+			linux,code = <BTN_2>;
+			gpios = <&gpio 7 GPIO_ACTIVE_LOW>;
```

Comment:
Isn't this a switch and we should define "linux,input-type = <EV_SW>;"?

Reply:
Oh, I missed it...I'll fix it, thank you!